wgpu-core/src/device/mod.rs

```diff
@@ -1288,6 +1293,7 @@ impl<G: GlobalIdentityHandlerFactory> Global<G> {
                 .iter()
                 .filter(|b| b.has_dynamic_offset())
                 .count(),
+            count_validator,
```

Comment:
We should actually validate this one here and return an error in `create_bind_group` if any of the limits are breached. See https://github.com/gpuweb/gpuweb/issues/409#issuecomment-649099520
Basically, it's of no use to try to allow such a bind group layout anyway, just brings more trouble.